spec/notifications-spec.coffee

```diff
@@ -263,6 +263,28 @@ describe "Notifications", ->
 
           runs ->
             expect(atom.notifications.getNotifications().length).toBe 0
+      describe "when an exception contains newline", ->
+        beforeEach ->
+          stack = """
+            TypeError: undefined is not a function
+              at Object.module.exports.Pane.promptToSaveItem [as defaultSavePrompt] (/Applications/Atom.app/Contents/Resources/app/src/pane.js:490:23)
+              at Pane.promptToSaveItem (/Users/someguy/.atom/packages/save-session/lib/save-prompt.coffee:21:15)
+              at Pane.module.exports.Pane.destroyItem (/Applications/Atom.app/Contents/Resources/app/src/pane.js:442:18)
+              at HTMLDivElement.<anonymous> (/Applications/Atom.app/Contents/Resources/app/node_modules/tabs/lib/tab-bar-view.js:174:22)
+              at space-pen-ul.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app/node_modules/archive-view/node_modules/atom-space-pen-views/node_modules/space-pen/vendor/jquery.js:4676:9)
+              at space-pen-ul.elemData.handle (/Applications/Atom.app/Contents/Resources/app/node_modules/archive-view/node_modules/atom-space-pen-views/node_modules/space-pen/vendor/jquery.js:4360:46)
+          """
+          detail = 'ok'
+
+          atom.notifications.addFatalError(stack, {detail, stack})
+          notificationContainer = workspaceElement.querySelector('atom-notifications')
+          fatalError = notificationContainer.querySelector('atom-notification.fatal')
+        it "title doesn't contain newline", ->
+          waitsForPromise ->
+            fatalError.getRenderPromise().then ->
+              issueTitle = fatalError.issue.getIssueTitle()
+          runs ->
+            expect(issueTitle).not.toContain "\n"
```

Comment:
@ungb: Thanks for adding this test. :zap:

Based on this test, I see that we don't want the issue title to contain a newline. That tells us a bit about what the issue title should _not_ look like. Can I talk you into saying what the issue title _should_ look like? I think that will provide additional clarity regarding the expected behavior. 

Right now, as far as I can tell, the issue title has two spaces between "function" and "at":

    TypeError: undefined is not a function  at Object.module.exports.Pane.promptToSaveItem [as defau...

Does that matches your expectations?